docs/reference/api/usb/USBHID.md

```diff
@@ -2,14 +2,14 @@
 
 <span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_h_i_d.png)<span>USBHID class hierarchy</span></span>
 
-You can use the USBHID class to send and receive messages over USB. For example, you can define your own protocol and communicate between your computer and Mbed with all capabilities of USB communication. To use USBHID, you need a script running on the host side (computer). For example, on a 32-bit Windows 7 machine, you can use [pywinusb](https://github.com/rene-aguirre/pywinusb).
+You can use the USBHID class to turn an Mbed board into an HID (Human Interface Device) that can send and receive messages over USB. For example, you can define your own protocol and communicate between your computer and Mbed with all capabilities of USB communication. To use USBHID, you need a script running on the host side (computer). For example, on a 32-bit Windows 7 machine, you can use [pywinusb](https://github.com/rene-aguirre/pywinusb). For convenience, there is a Python script attached below that uses pywinusb that can be run on the host computer to send and receive data from the Mbed board.
```

Comment:
`on a 32-bit Windows 7 machine` is a bit specific. Could this be replaced with just `Windows machine`?

Reply:
fixed